NorthstarDLL/plugins/plugin_abi.h

```diff
@@ -61,14 +49,22 @@ struct SquirrelFunctions
 	sq_getassetType __sq_getasset;
 	sq_getuserdataType __sq_getuserdata;
 	sq_getvectorType __sq_getvector;
+	sq_getthisentityType __sq_getthisentity;
+	sq_getobjectType __sq_getobject;
+
+	sq_stackinfosType __sq_stackinfos;
```

Comment:
Was there a reason to move these entries around? Grouping them with similar related entries maybe? ^^